datacats/environment.py

```diff
@@ -23,7 +23,8 @@
 from datacats.template import ckan_extension_template
 from datacats.scripts import (WEB, SHELL, PASTER, PASTER_CD, PURGE,
     RUN_AS_USER, INSTALL_REQS, CLEAN_VIRTUALENV, INSTALL_PACKAGE,
-    COMPILE_LESS, DATAPUSHER, INSTALL_POSTGIS, ADJUST_DEVINI)
+    COMPILE_LESS, DATAPUSHER, INSTALL_POSTGIS, ADJUST_DEVINI,
```

Comment:
*(Bikeshedding like there is no tomorrow here. Brace yourselves)*

Since we are importing so many items from this namespace and use each one only once, maybe we should start importing this explicitly with:
```
import datacats.scripts
```

In general, I am in favour of this style in most cases. It is much more explicit and improve code's readibility and comprehension.
When you encounter:
```
blah=datacats.scripts.UPDATE_ADD_ADMIN
```
it is clear right away where this thing is defined and is coming from. 

On the other hand, this:
```
blah=UPDATE_ADD_ADMIN
```
will require an extra lookup to figure out.

I think this form
```
from blah import blahblah
```
only makes sense when `blahblah` is used quite a few times (4 or more) or at least a part of something presumed really well known (like widely used parts of standard library or some key library object).